src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -1772,7 +1772,7 @@ public static IntPtr SecureStringToBSTR(SecureString s)
 #if FEATURE_COMINTEROP
             return s.MarshalToBSTR();
 #else
-            throw new PlatformNotSupportedException();
```

Comment:
This is bug/inconsistency. We have implemented BSTR on Unix, even though it is COM interopy. Mono took the same approach. We should remove the FEATURE_COMINTEROP ifdef around this one and make it work on Unix - to be consistent with other methods that take BSTRs.